napari/layers/image/image.py

```diff
@@ -778,7 +778,7 @@ def _get_slice(request: _ImageSliceRequest) -> _ImageSliceResponse:
             data, thumbnail, tile_to_data = Image._get_slice_data_multi_scale(
                 slice_indices, request
             )
-            full_transform = tile_to_data.compose(full_transform)
+            full_transform = full_transform.compose(tile_to_data)
```

Comment:
I don't know what happened here... Looking at [this](https://github.com/andy-sweet/napari/pull/21/files#diff-9e4cd2702cfbb10e85a517ed53426c08d163353b58aab0e13609b4375c1e9780R781) it seems mine is the latest?

Reply:
Yeah, I got the order wrong initially, which may for a fun experience with multi-scale images... I guess you merged this change in at some point and made this comment, but this diff is shown as outdated and doesn't appear in the files changed tab, so should be fine.